util/config/config.go

```diff
@@ -50,39 +50,43 @@ var (
 		SyncBatchWindowSize:       1024,
 		SyncBlockHeadersBatchSize: 256,
 		SyncBlocksBatchSize:       8,
+		RPCReadTimeout:            5,
```

Comment:
What about making this same as RPCWriteTimeout? Most home Internet has a much lower upload speed than download speed.

Reply:
@yilunzhang For Https, the WriteDeadline also covers the packets written as part of the TLS handshake. It means that WriteTimeout ends up including the header read and the first byte wait.

![](https://blog.cloudflare.com/content/images/2016/06/Timeouts-001.png)

https://blog.cloudflare.com/the-complete-guide-to-golang-net-http-timeouts/